modules/@apostrophecms/piece-type/index.js

```diff
@@ -457,7 +458,7 @@ module.exports = {
           return self.apos.modules['@apostrophecms/job'].runBatch(
             req,
             req.body._ids,
-            async function(req, id) {
+            async function (req, id) {
```

Comment:
here's another

Reply:
Serched for and fixed additional instances.